api/server/structs/endpoints_beacon.go

```diff
@@ -172,6 +172,11 @@ type GetAttesterSlashingsResponse struct {
 	Data []*AttesterSlashing `json:"data"`
 }
 
+type GetAttesterSlashingsV2Response struct {
```

Comment:
This is a different approach than what you did for https://github.com/prysmaticlabs/prysm/pull/14481/files#diff-b7de0123b995294eb1e799afb080f62c9a4a08195f9ed1f387a9071922b1ec97. Can you do the same here?